### ISSUE TITLE
Doesn't work on the macOS Terminal

### ISSUE THREAD

**Issue by Chiroyce1** [Sid72020123/pyTextColor#2]
![image](https://user-images.githubusercontent.com/97374054/150671248-915e38bf-27cd-4448-bff0-539ddd4e5da4.png)

Tested on macOS Big Sur 11.6.2, with the default Terminal Version 2.11 (440)

Colorama works though.

<img width="243" alt="image" src="https://user-images.githubusercontent.com/97374054/150671238-0730a448-d5ee-4c34-a2fc-b66a7faa9a82.png">

**Comment by Sid72020123** [comment]
Oh. Maybe I should find a way to make this work in MacOS...

**Comment by imdevarsh** [comment]
Let me try...

Edit: Works in [Hyper](https://github.com/vercel/hyper) but not in default terminal.

**Comment by Sid72020123** [comment]
Oh

**Comment by Sid72020123** [comment]
Colorama converts ANSI to win32 chars on  Windows. That's why it works in Windows Command Prompt:
![image](https://user-images.githubusercontent.com/70252606/150672455-057c6673-6780-4ca8-b214-7778be7d7833.png)

**Comment by Sid72020123** [comment]
Found a trick  that makes it work in Windows Command Prompt:
![image](https://user-images.githubusercontent.com/70252606/150673007-3eed14c3-5a90-4750-8dbc-55cd5a63eec3.png)

You just need to write the code like:
```python
import os

os.system("")

import pyTextColor

pytext = pyTextColor.pyTextColor()

text = pytext.format_text(text="This text will be printed out in yellow color with red background", color="#ff0000",
                          bgcolor="#FFFF00")  # HEX values

print(text)  # Print the text
```
I found that using this method, styles doesn't work :(

**Comment by Sid72020123** [comment]
Can any macOS user try the above code to check if it works in the default Terminal? I am on Windows OS :(

**Comment by Sid72020123** [comment]
Also using this code
```python
from colorama import init
init()

import pyTextColor

pytext = pyTextColor.pyTextColor()

text = pytext.format_text(text="Test", color="cyan",
                          bgcolor="black", text_style="underline")  # HEX values

pytext.test_print(text)
print(text)

```
The above code works in Windows Command Prompt. Can anyone test it in macOS?

**Comment by Sid72020123** [comment]
Only the styles doesn't work :(

**Comment by Sid72020123** [comment]
tinted doesn't work too :(

**Comment by Chiroyce1** [comment]
<img width="835" alt="image" src="https://user-images.githubusercontent.com/97374054/150675125-5fa334af-a320-4088-9d53-05416fd2019a.png">

**Comment by Chiroyce1** [comment]
> The above code works in Windows Command Prompt. Can anyone test it in macOS?

`AttributeError: 'pyTextColor' object has no attribute 'test_print'`

**Comment by Sid72020123** [comment]
> > The above code works in Windows Command Prompt. Can anyone test it in macOS?
> 
> `AttributeError: 'pyTextColor' object has no attribute 'test_print'`

Oh Sorry. Remove that `pytext.test_print(text)` function. I am really very sorry :(

**Comment by Chiroyce1** [comment]
> Oh Sorry. Remove that `pytext.test_print(text)` function. I am really very sorry :(

It's fine - you don't have to be sorry 😄 - it works now

<img width="443" alt="image" src="https://user-images.githubusercontent.com/97374054/150681202-16d7ea47-e948-41bb-8737-38c821a8c0b6.png">


But hex colors don't work

**Comment by Sid72020123** [comment]
Yes HEX colors won't work because they can only be displayed on some Terminals :( So that's why colorama didn't added HEX Colors. LOL. Also styles won't work on normal terminals.
I tested that this works in Replit, PyCharm, Windows Powershell.
Thank you so much Chiroyce for helping me to test this all.

**Comment by kccuber-scratch** [comment]
> Windows Powershell

the powershell that you tested this in (from the screenshot in the other issue) is actually windows terminal. try testing it in the actual app called windows powershell (yes it's a seperate app)

**Comment by Sid72020123** [comment]
Oh! Sorry I should have wrote "Windows Terminal". Is "Windows Powershell" and "Windows Command Prompt" the same?

**Comment by Sid72020123** [comment]
I found a solution: https://stackoverflow.com/questions/51680709/colored-text-output-in-powershell-console-using-ansi-vt100-codes
I won't try it now because I'm a little busy...

**Comment by Sid72020123** [comment]
> Oh! Sorry I should have wrote "Windows Terminal". Is "Windows Powershell" and "Windows Command Prompt" the same?

NVM Found it.
